tests/test_spidermiddleware.py

```diff
@@ -290,10 +289,7 @@ def process_spider_output(self, response, result, spider):
 
 class ProcessSpiderOutputCoroutineMiddleware:
     async def process_spider_output(self, response, result, spider):
-        results = []
-        for r in result:
-            results.append(r)
-        return results
+        return result
```

Comment:
Interesting. It is technically not the same, since result is not a list but an iterator. On the other hand, in this specific case it is most likely OK.

Reply:
I rewrote this myself, the linter suggests a copy via `list(result)`.

Reply:
(Well, not a copy, as it's not necessarily a list)